### ISSUE TITLE
Cheat files

### ISSUE THREAD

**Issue by krabobmkd** [krabobmkd/amigamame#4]
There's a thing called cheat files and a way to activate cheats in game, no idea if it works and how, but will be done for beta 2.

**Comment by Jeff-Bootonic** [comment]
Cheats work in conjunction with the cheat.dat file, which must be copied separately into the directory. Then there are the history and highscore.dat files, which also need to be copied separately and can be displayed in the MAME menu. I had to modify the cheat option in MAME 0.37 because the checks were disabled and the .dat file wasn't being read. Perhaps this is a way to activate them, but I am sure you will find a better way than mine.

**Comment by Jeff-Bootonic** [comment]
I checked it out and it turns out that the cheat.dat needs to go into the user directory, and it already works now.

**Comment by krabobmkd** [comment]
yes , I've been confused by mame's 3 level file management. In beta 2 it will be reachable with an absolute path in coinfiguration, out of user/ .

**Comment by krabobmkd** [comment]
corrected in beta2
